pkg/cmd/server/origin/master_config.go

```diff
@@ -293,7 +293,7 @@ var (
 		"ProjectRequestLimit",
 		"OriginNamespaceLifecycle",
 		"PodNodeConstraints",
-		"JenkinsBootstrapper",
+		//"JenkinsBootstrapper",
```

Comment:
Removing it this way makes it essentially impossible to turn back on.  If you want it off by default, use the code here: https://github.com/openshift/origin/blob/master/pkg/cmd/server/start/admission.go#L43